third_party/weval/weval.h

```diff
@@ -0,0 +1,482 @@
+#pragma once
```

Comment:
Do we need to include a copy of the weval license in this folder?

Reply:
Done!